pkg/config/linters_exclusions.go

```diff
@@ -0,0 +1,41 @@
+package config
+
+import (
+	"fmt"
+)
+
+const (
+	DefaultExclusionComments             = "comments"
+	DefaultExclusionStdErrorHandling     = "stdErrorHandling"
+	DefaultExclusionCommonFalsePositives = "commonFalsePositives"
+	DefaultExclusionLegacy               = "legacy"
+)
+
+const excludeRuleMinConditionsCount = 2
+
+type LinterExclusions struct {
+	Generated   string        `mapstructure:"generated"`
+	WarnUnused  bool          `mapstructure:"warn-unused"`
+	Default     []string      `mapstructure:"default"`
+	Rules       []ExcludeRule `mapstructure:"rules"`
+	Paths       []string      `mapstructure:"paths"`
+	PathsExcept []string      `mapstructure:"paths-except"`
+}
+
+func (e *LinterExclusions) Validate() error {
```

Comment:
Should we validate the values in `Default` as well? Looks like we blindly pass it to `getDefaultLintersExclusions` which assumes the values will exist in `defaultLintersExclusions`?